src/wallet.ts

```diff
@@ -9,59 +9,59 @@ import * as sigUtil from 'eth-sig-util';
 import { ethErrors } from 'eth-rpc-errors';
 import { Block } from './utils/cache';
 
-interface TransactionParams {
+export interface TransactionParams {
   from: string;
 }
 
-interface MessageParams extends TransactionParams {
+export interface MessageParams extends TransactionParams {
   data: string;
 }
 
-interface TypedMessageParams extends MessageParams {
+export interface TypedMessageParams extends MessageParams {
   version: string;
 }
 
-interface WalletMiddlewareOptions {
+export interface WalletMiddlewareOptions {
   getAccounts: (req: JsonRpcRequest<unknown>) => Promise<string[]>;
   processDecryptMessage?: (
     msgParams: MessageParams,
     req: JsonRpcRequest<unknown>,
-  ) => Promise<Record<string, unknown>>;
+  ) => Promise<string>;
```

Comment:
`Returns` is a string type
https://docs.metamask.io/guide/rpc-api.html#eth-decrypt